### PR TITLE
Front and End Matter Updates

### LINKED ISSUE #1: Fix broken slides
Add venv section back into slides.

Fix front and end matter so that they include all of us and are relevant to the summer school.

### PR DESCRIPTION
**Pull request by jatkinson1000**
Closes #1 

Adds Amy and Marion as authors, removes personal info

Adds section on code clinics at the end

Fixes broken/inappropriate links and images

### CODE REVIEW

**Review by AmyOctoCat — APPROVED** [review]
LGTM :)

File: slides/python.qmd [hunk 1]
```diff
@@ -236,6 +232,22 @@ Get in touch:
...
+{{< fa brands github >}} \ [AmyOctoCat](https://github.com/AmyOctoCat)
+
+\
+
+{{< fa pencil >}} \ Marion Weinzierl
+
+{{< fa solid envelope >}} \ [mw925[AT]cam.ac.uk](mailto:mw925@cam.ac.uk)
+
+{{< fa brands github >}} \ [MarionBWeinzierl](https://github.com/MarionBWeinzierl)
+
```
Comment (MarionBWeinzierl) [inline slides/python.qmd:253]:
```suggestion

{{< fa brands mastodon >}} \ [\@MarionBWeinzierl\@mast.hpc.social](https://mast.hpc.social/@MarionBWeinzierl)
```

Comment (MarionBWeinzierl) [reply slides/python.qmd:253]: If we want to do Mastodon, this is mine

**Review by MarionBWeinzierl — APPROVED** [review]
This looks good to me. We can merge this now, and decide later whether we want to add more infor.

**Pull request merged**